test/Libraries/RevitNodesTests/Elements/StructuralFramingTests.cs

```diff
@@ -66,7 +66,16 @@ public void TestLocationOfStructuralFraming()
 
             var beam = e as StructuralFraming;
             Assert.NotNull(beam);
-            var curve = beam.Location;
+            var loc = beam.Location;
+
+            var pt = loc as Point;
+            if (pt != null)
+            {
+                Assert.NotNull(pt);
```

Comment:
This Assert is not useful because of IF condition. I think you want to Assert.IsNull(pt) and Assert.IsNotNull(curve)